### PR TITLE
Update Norwegian translations

### PR DESCRIPTION
**Pull request by yookoala**
* Redid works of @thor-the-norseman on the current master branch.
  * Updated no.json
  * Updated activerecord.no.yml
  * Updated no.yml
  * Updated simple_form.no.yml
* Some fixes on the current data (fix UTF-8 error, remove-unused).

Resolves #2676.

### CODE REVIEW

**Review by ykzts — APPROVED** [review]

File: config/locales/activerecord.no.yml [hunk 1]
```diff
@@ -1,12 +1 @@
...
-    errors:
-      models:
-        account:
-          attributes:
-            username:
-              invalid: bare bokstaver, tall og understreker
-        status:
-          attributes:
-            reblog:
-              taken: av status eksisterer allerede
```
Comment (ykzts) [inline config/locales/activerecord.no.yml]: This keys is target of [ignore_unused](https://github.com/tootsuite/mastodon/blob/ef226a6f223ad3bdf662efb0ed75694b3f0bfb56/config/i18n-tasks.yml#L46).

**Review by ykzts — CHANGES_REQUESTED** [review]

File: config/locales/activerecord.no.yml (hunk 1 above)
Comment (yookoala) [inline config/locales/activerecord.no.yml]: Should I remove this?

Comment (ykzts) [inline config/locales/activerecord.no.yml]: No, This file is also in other languages, so you do not need to delete it.

Comment (yookoala) [inline config/locales/activerecord.no.yml]: Restored. See 6d14d9e.

Comment (ykzts) [inline config/locales/activerecord.no.yml]: good!

**Comment by ykzts** [comment]
Uhnn...? CI failed.

File: config/locales/activerecord.no.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+no:
```
Comment (ykzts) [inline config/locales/activerecord.no.yml]: Please escape like `'no':`. It is recognized as false in YAML.

**Review by ykzts — CHANGES_REQUESTED** [review]

**Comment by yookoala** [comment]
@ykzts The current master HEAD, which this PR is based on, has other errors.

File: config/locales/activerecord.no.yml (hunk 2 above)
Comment (yookoala) [inline config/locales/activerecord.no.yml]: Fixed. See e0ed31c.

**Comment by ykzts** [comment]
@yookoala I think that it will probably be repaired with e0ed31c. In my environment, it worked fine.

**Review by ykzts — APPROVED** [review]

**Pull request merged**